docs/help.md

```diff
@@ -0,0 +1,27 @@
+### Commands
+`bot adapter` - Reply with the adapter
+
+`bot create|close milestone all "title"` - Create or close all milestones listed in `config.json`
+`bot create|close milestone "title" in "myorg/myrepo"` - Create and close milestone in the specific github repo
+  - *title* - can accept the following values:
+    - any date in the format of `DD MMM YYYY`
+    - any string
+    For example: "13 Jan 2017" or "This is Backlog"
+    
+`bot issue "title" about "body" in "myorg/myrepo"` - Create an issue on github
+  - *title* - the title of the issue
+  - *body* - the contents of the issue
+
+`bot links` - Return gdocs url for tag `+link`
```

Comment:
Gdocs or Google Docs

Reply:
Actually it's Google Spreadsheets as much as I remember

Reply:
Google Docs is a free Web-based application in which documents and spreadsheets can be created. We can use google spreadsheet to clarify

Reply:
Yes please, lets use spreadsheet